lib/api.js

```diff
@@ -125,6 +129,8 @@ module.exports = (function () {
 
   var mbaas_url = url.parse(mbaas_protocol + "://" + mbaas_host);
 
+  var logLevel = parseInt(process.env.FH_LOG_LEVEL) || 2;
```

Comment:
This is pretty neat, but one thing comes to mind. If we wanted to set this in the Studio I think it would not work, since "FH_" is treated as a reserved name?